test/format/bulk.c

```diff
@@ -103,7 +103,7 @@ table_load(TABLE *base, TABLE *table)
         wt_wrap_open_cursor(session, base->uri, NULL, &base_cursor);
 
     /* No bulk load with custom collators, insertion order won't match collation order. */
-    is_bulk = TV(BTREE_REVERSE) == 0;
+    is_bulk = TV(BTREE_REVERSE) == 0 && TV(DISAGG_ENABLED) == 0;
```

Comment:
I think it would be better to break this out, e.g. a new line with `is_bulk &= TV(DISAGG_ENABLED) == 0`. That way the comment above the bulk load still "matches" the code.